tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/effects/debugger_effects.ts

```diff
@@ -176,6 +180,28 @@ export class DebuggerEffects {
     );
   }
 
+  /**
+   * Load dtypes info when debugger plugin is loaded.
+   */
+  private loadDtypesMap(prevStream$: Observable<void>) {
+    return prevStream$.pipe(
+      withLatestFrom(this.store.select(getDtypesMapLoaded)),
+      filter(([, dtypesMapLoadState]) => {
+        return dtypesMapLoadState.state !== DataLoadState.LOADING;
+      }),
+      tap(() => this.store.dispatch(dtypesMapRequested())),
+      mergeMap(() => {
+        return this.dataSource.fetchDtypesMap().pipe(
+          tap((dtypesMapResponse: DtypesMapResponse) => {
+            this.store.dispatch(dtypesMapLoaded(dtypesMapResponse));
+          }),
+          map(() => void null)
+          // TODO(cais): Add catchError() to pipe.
```

Comment:
What do you think should happen when the failure happens? Does entire UI break when this happens?